server/admin/http/handler.go

```diff
@@ -394,3 +395,42 @@ func decodePeerID(id string, w http.ResponseWriter) (peer.ID, bool) {
 	}
 	return peerID, true
 }
+
+var healthCheckMH multihash.Multihash
+var healthCheckValue indexer.Value
+
+func init() {
+	provider, err := peer.Decode("12D3KooWBUNzpAz1Jfvnaag1nHBi6gbG5Q1BQm8kCfcpmt7bGKa6")
+	if err != nil {
+		panic(err.Error())
+	}
+	healthCheckMH = multihash.Multihash("2DrjgbFdhNiSJghFWcQbzw6E8y4jU1Z7ZsWo3dJbYxwGTNFmAj")
+	healthCheckValue = indexer.Value{
+		ProviderID:    provider,
+		ContextID:     []byte(healthCheckMH),
+		MetadataBytes: []byte("healthcheck-metadata"),
```

Comment:
Use a valid metadata to avoid lower level errors in case this is checked for correctness.